### PR TITLE
Make candidate validation timeouts configurable

### PR DESCRIPTION
**Pull request by rphmeier**
and use a shorter timeout for backing as compared to approval and dispute participation.

### CODE REVIEW

File: node/subsystem-types/src/messages.rs [hunk 1]
```diff
@@ -130,6 +133,8 @@ pub enum CandidateValidationMessage {
 		ValidationCode,
 		CandidateDescriptor,
 		Arc<PoV>,
+		Duration,
+		/// Execution timeout
```
Comment (pepyakin) [inline node/subsystem-types/src/messages.rs:137]: rustfmt considers comments on the same related to the next line...

File: node/subsystem-types/src/messages.rs [hunk 2]
```diff
@@ -114,6 +115,8 @@ pub enum CandidateValidationMessage {
 	ValidateFromChainState(
 		CandidateDescriptor,
 		Arc<PoV>,
+		Duration,
+		/// Execution timeout
```
Comment (pepyakin) [inline node/subsystem-types/src/messages.rs:119]: .

**Review by pepyakin — APPROVED** [review]
Looks good modulo the test compilation error

File: node/subsystem-types/src/messages.rs (hunk 1 above)
Comment (rphmeier) [reply node/subsystem-types/src/messages.rs:137]: I see, so fmt moved this around.

**Comment by rphmeier** [comment]
bot merge

**Comment by rphmeier** [comment]
bot merge cancel

**Review by eskimor — APPROVED** [review]

**Pull request merged**